### DISCUSSION TITLE
两个显示问题，没有找到文档 || Two display problems, no documents found

### DISCUSSION THREAD

**Discussion by hulang** [walinejs/waline#1965] [Q&A]
### Describe the bug | 问题描述

1、我网站是通过QQ登录，才能评论，能不能没有登录的时候，不显示：评论框，但是显示评论列表。
2、时间显示问题，我不想显示：X 天前，X 小时前 类似这样的，我想直接显示：2023-05-09 16:40:15  这样的。。

初始化参数里面，能调整吗？

### Website URL | 问题网站

https://waline.js.org/

### Where your waline deploy? | 服务部署在哪里？

Other

### Where your comment data store? | 数据存储在哪里？

MySQL

**Comment by lizheming** [comment]
1. 可以外部通过获取登录信息状态直接通过 CSS 隐藏评论框，登录信息获取可以参考下这个 demo https://github.com/orgs/walinejs/discussions/1557
2. 这个应该不支持，需要的话可以自己自行 fork 修改

---
1. The comment box can be hidden externally through CSS directly by obtaining the status of the login information. For obtaining the login information, please refer to this demo https://github.com/orgs/walinejs/discussions/1557
2. This should not be supported, if necessary, you can fork and modify it yourself

**Reply by hulang** [reply]
好的，谢谢了。。

---
Ok, thanks. .
